packages/skygear-core/src/error.ts

```diff
@@ -1,26 +1,95 @@
 import { JSONObject } from "./types";
 
+/**
+ * @public
+ */
+export const SkygearErrorNameUnexpectedError = "UnexpectedError" as const;
```

Comment:
Can we consolidate the error names in an object? e.g.
```typescript
export const SkygearErrorNames = {
    UnexpectedError: 'UnexpectedError',
    // ...
};
```

Reply:
OK.